plugins/auth/install.php

```diff
@@ -6,6 +6,7 @@
 rex_config::set('ycom', 'auth_request_psw', 'rex_ycom_auth_psw');
 rex_config::set('ycom', 'auth_request_id', 'rex_ycom_auth_id');
 rex_config::set('ycom', 'auth_request_stay', 'rex_ycom_auth_stay');
+rex_config::set('ycom/auth', 'login_tries', 10);
```

Comment:
ich vermute der key ist falsch für die logik in https://github.com/yakamara/redaxo_ycom/pull/124/files#diff-f7dcd3a67d9832ec712d36a8d43aaab8R111

ich denke es muss `rex_config::set('ycom', 'auth_login_tries', 10);` heißen